automated_analysis.py

```diff
@@ -158,16 +158,17 @@ def coding_plans_to_analysis_configurations(coding_plans):
             limit_per_code=100
         )
 
-    if pipeline_configuration.automated_analysis.traffic_labels is not None:
-        log.info("Exporting traffic analysis...")
-        with open(f"{automated_analysis_output_dir}/traffic_analysis.csv", "w") as f:
-            traffic_analysis.export_traffic_analysis_csv(
-                messages, CONSENT_WITHDRAWN_KEY,
-                coding_plans_to_analysis_configurations(PipelineConfiguration.RQA_CODING_PLANS),
-                SENT_ON_KEY,
-                pipeline_configuration.automated_analysis.traffic_labels,
-                f
-            )
+    # TODO: Re-enable once supporting different sent_on_keys and the traffic timestamps are fixed.
```

Comment:
This is now fixed best to un comment this here before merging

Reply:
The automated analysis, and some other parts of the pipeline, assumed that all rqa messages have a field called "sent_on" that would be the same, which isn't the case here. The easiest way out of this will be just to rename the time keys for the follow-ups to "sent_on", which I can do now.